Source/Scene/TileMapServiceImageryProvider.js

```diff
@@ -315,13 +315,7 @@ define([
          */
         tileWidth : {
             get : function() {
-                //>>includeStart('debug', pragmas.debug);
-                if (!this._ready) {
```

Comment:
You still need these checks.  But instead of `!this._ready` use `!defined(this._imageryProvider)`.